tests/components/websocket_api/test_commands.py

```diff
@@ -695,14 +695,21 @@ async def test_get_services(
     hass: HomeAssistant, websocket_client: MockHAClientWebSocket
 ) -> None:
     """Test get_services command."""
+    local_services = hass.services.async_services()
+
     for id_ in (5, 6):
         await websocket_client.send_json({"id": id_, "type": "get_services"})
 
         msg = await websocket_client.receive_json()
         assert msg["id"] == id_
         assert msg["type"] == const.TYPE_RESULT
         assert msg["success"]
-        assert msg["result"] == hass.services.async_services()
+
+        for domain_name in local_services:
```

Comment:
If I understand correctly this is now asserting that the each service is contained in`msg["result"]`, but there might be extra services or domains in `msg["result"]` and the test would still pass whereas previously it would have failed

Why did this need to be modified?